Formula/qt.rb

```diff
@@ -24,39 +24,35 @@ class Qt < Formula
     sha256 "25c4a693c787860b090685ac5cbeea18128d4d6361eed5b1bfed1b16ff6e4494" => :mojave
   end
 
-  keg_only "Qt 5 has CMake issues when linked"
+  keg_only "Qt has CMake issues when linked"
```

Comment:
This issue can be solved by adding some configuration options.

Reply:
Which ones. Please provide suggestions to the code

Reply:
See below, I provide cmake configuration options(configure script may has same options), the important points are ``INSTALL_XXXDIR`` and for ``CMAKE_STAGING_PREFIX`` , See https://cmake.org/cmake/help/latest/variable/CMAKE_FIND_NO_INSTALL_PREFIX.html